server/sonar-server/src/main/resources/org/sonar/server/platform/ws/example-migrate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "operational": "false",
+  "state": "MIGRATION_RUNNING",
+  "message": "Database migration is running",
+  "createdAt": "2015-02-23T18:54:23+0100"
```

Comment:
well spotted

this is an example of a practice that I think should be enforced when writing WS: a unit test should use the example as expected JSON.

this way, examples will always be consistent and accurate